internal/utils.go

```diff
@@ -189,6 +190,39 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]interface{
 			if isPolicyTypeManifest {
 				policyTemplate := map[string]interface{}{"objectDefinition": manifest}
 
+				objectTemplatesRaw, found, _ := unstructured.NestedString(manifest, "object-templates-raw")
+				if found {
+					policyGenNum = len(policyTemplates) + 1
+					var name string
+
+					if policyGenNum > 1 {
+						name = fmt.Sprintf("%s%d", policyConf.Name, policyGenNum)
+					} else {
+						name = policyConf.Name
+					}
+
+					policyTemplate = map[string]interface{}{
+						"objectDefinition": map[string]interface{}{
+							"apiVersion": policyAPIVersion,
+							"kind":       configPolicyKind,
+							"metadata": map[string]interface{}{
+								"name": name,
+							},
+							"spec": map[string]interface{}{
+								"object-templates-raw": objectTemplatesRaw,
```

Comment:
I think doing it this way might miss some configurations, like dependencies and diff logging. Rather than handling `object-templates-raw` as a policy type, could we instead somehow pass it through to `buildPolicyTemplates()` directly and inject it there?
https://github.com/open-cluster-management-io/policy-generator-plugin/blob/a097bd7881bb6ab72032449e316b1c691150f094/internal/utils.go#L419

There might be other/better ways, but I'm thinking you could rename the `objectTemplates` argument to `policySpec` and set either the `object-templates-raw` or `object-templates` key directly as part of it. Then set remediationAction and severity before setting the `spec` directly:
```golang
policySpec["remediationAction"] = policyConf.RemediationAction
policySpec["severity"] = policyConf.Severity
...
"spec": policySpec,
```

Reply:
Thanks, I'll look into it.